R/texreg.R

```diff
@@ -6,13 +6,13 @@
 # screenreg function
```

Comment:
I'm not exactly sure what plotreg does (I don't use it), but it looks quite different: https://github.com/leifeld/texreg/blob/master/R/plotreg.R#L214

Unfortunately, I don't think I can justify the time given that I won't use it...